src/styles/main.scss

```diff
@@ -248,6 +248,11 @@ h1, h2, h3, h4, h5, h6 {
   animation: rainbow-text 2s infinite;
 }
 
+.required:after {
```

Comment:
Wait, would this add a red star to every required thing? I would put this at a different PR since this impacts the entire frontend

Reply:
Ah oops did not mean to push that yet, since it only works for like half of the cases 😅